sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLTgacSubmissionDAO.java

```diff
@@ -381,7 +381,7 @@ public SubmissionImpl mapRow(ResultSet rs, int rowNum) throws SQLException {
               runDAO.listBySequencerPartitionContainerId(partition.getSequencerPartitionContainer().getId()));
           // if there is 1 run for the flowcell/container, sets the run for that container to the first on on the list
           if (runs.size() == 1) {
-            partition.getSequencerPartitionContainer().setRun(runs.get(0));
+            partition.getSequencerPartitionContainer().addRun(runs.get(0));
```

Comment:
I think this should be a set, rather than an add, since the container could already have runs... Not really sure though